src/app/component/eco-news/components/news-preview-page/news-preview-page.component.ts

```diff
@@ -13,12 +12,12 @@ import { Subscription } from 'rxjs';
   styleUrls: ['./news-preview-page.component.scss']
 })
 export class NewsPreviewPageComponent implements OnInit, OnDestroy {
-  private images = singleNewsImages;
-  private previewItem: FormGroup;
-  private actualDate = new Date();
-  private userName: string;
+  public images = singleNewsImages;
+  public previewItem: FormGroup;
+  public actualDate = new Date();
+  public userName: string;
   private userNameSub: Subscription;
-  private isPosting = false;
+  public isPosting = false;
```

Comment:
Please put all public first and then all private

Reply:
> Please put all public first and then all private

Fixed